src/main/java/it/pagopa/interop/probing/response/updater/tracing/config/XRayMDCConfig.java

```diff
@@ -1,24 +1,21 @@
 package it.pagopa.interop.probing.response.updater.tracing.config;
 
+import java.util.UUID;
 import org.aspectj.lang.JoinPoint;
 import org.aspectj.lang.annotation.Aspect;
 import org.aspectj.lang.annotation.Before;
+import org.slf4j.MDC;
 import org.springframework.stereotype.Component;
-import com.amazonaws.xray.AWSXRay;
 import it.pagopa.interop.probing.response.updater.util.logging.LoggingPlaceholders;
 
 @Aspect
 @Component
 public class XRayMDCConfig {
 
-  @Before("execution(* it.pagopa.interop.probing.response.updater.service.impl..*(..))")
-  public void beforeQueryOperation(JoinPoint joinPoint) {
-    AWSXRay.beginSubsegment(
-        LoggingPlaceholders.AURORA_SUBSEGMENT_NAME + " " + joinPoint.getSignature().getName());
-  }
+  @Before("execution(* it.pagopa.interop.probing.response.updater.consumer..*(..))")
+  public void beforeConsumer(JoinPoint joinPoint) {
+    MDC.put(LoggingPlaceholders.TRACE_ID_PLACEHOLDER,
+        "- [CID= " + UUID.randomUUID().toString().toLowerCase() + "]");
```

Comment:
shouldn't this be the XRay tracing id instead of a randomUUID?

Reply:
this is the existing configuration because during the development I was told that this was a temporary value, and it would have been changed accordingly when the tracing would have been completed.
A random value for a correlation id is not useful